features/gba-dev/kinderen.feature

```diff
@@ -0,0 +1,146 @@
+# language: nl
+
+@post-assert
+Functionaliteit: Kinderen van een persoon raadplegen
+  Van een persoon worden -indien gevraagd met de fields parameter- de kinderen geleverd.
+  Dit bevat enkele identificerende eigenschappen van de kinderen.
+
+  Rule: de actuele gegevens van kinderen worden geleverd
+
```

Comment:
Is het in een dev scenario (en misschien ook wel in de functionele scenario's)  een idee om met de simpelste situaties te beginnen ? 
Wat gebeurt er als de persoon geen kinderen heeft en wat gebeurt er als de persoon gewoon een (of meer) kinderen heeft zonder dat daar iets byzonders mee is ? 

Hier worden direct allerlei complexere scenario's geschetst.

Reply:
Daar kan ik me wel in vinden maar daar hoeft het mergen van deze PR niet op te wachten. Dat kan ook in een aparte PR.

Reply:
Het is wel goed om beide scenario's die Johan voorstelt toe te voegen. Wel graag in een nieuwe PR.